tests/safeds_stubgen/stubs_generator/__snapshots__/test_generate_stubs.ambr

```diff
@@ -0,0 +1,152 @@
+# serializer version: 1
+# name: test_class_attribute_creation
+  '''
+  package test_stub_generation.attribute_module
+  
+  class A {}
+  
+  class B {
+  	attr type_hint_public: Int
+  	attr no_type_hint_public
+  	attr object_attr: A
+  	// Todo Tuple types are not allowed in SafeDS
+  	attr tuple_attr_1: Tuple
+  	// Todo Tuple types are not allowed in SafeDS
+  	attr tuple_attr_2: Tuple<union<String, Int>>
+  	// Todo Tuple types are not allowed in SafeDS
+  	attr tuple_attr_3: Tuple<String, Int>
+  	attr list_attr_1: List<Any>
+  	attr list_attr_2: List<union<String, A>>
+  	// Todo List type has to many type arguments
+  	attr list_attr_3: List<String, A>
+  	// Todo List type has to many type arguments
+  	attr list_attr_4: List<String, union<A, Int>>
+  	attr dict_attr_1: Map<Any, Any>
+  	attr dict_attr_2: Map<String, Int>
+  	attr dict_attr_3: Map<union<String, Int>, A?>
+  	attr bool_attr: Boolean
+  	attr none_attr: Nothing?
+  	attr flaot_attr: Float
+  	attr int_or_bool_attr: union<Int, Boolean>
+  	attr str_attr_with_none_value: String
+  	attr mulit_attr_1
+  	attr mulit_attr_3
+  	attr mulit_attr_5
+  	attr mulit_attr_6
+  	attr mulit_attr_7
+  	attr mulit_attr_8
+  	attr init_attr: Boolean
+  }
+  
+  '''
+# ---
+# name: test_class_creation
+  '''
+  package test_stub_generation.class_module
+  
+  class A {}
+  
+  class B(a: Int, b: A?) sub A {
+  	fun f()
+  }
+  
+  class C sub A, B {
+  	attr attr_1: Int
+  	attr attr_2: Int
+  
+  	fun f1()
+  }
+  
+  class D {
+  	class E {
+  		class F {		}
+  	}
+  }
+  
+  '''
+# ---
+# name: test_enum_creation
+  '''
+  package test_stub_generation.enum_module
+  
+  from `enum` import Enum
+  from `enum` import IntEnum
+  from `enum` import Enum as _Enum
+  
+  enum EnumTest {
+  	ONE
+  	TWO
+  	THREE
+  	FOUR
+  	FIVE
+  	SIX
+  	SEVEN
+  	EIGHT
+  	NINE
+  }
+  
+  enum EnumTest2 {
+  	TEN
+  }
+  
+  enum EnumTest3 {
+  	ELEVEN
+  }
+  
+  enum EmptyEnum {}
+  
+  '''
+# ---
+# name: test_function_creation
+  '''
+  package test_stub_generation.function_module
+  
+  fun public_no_params_no_result()
+  
+  fun params(i: Int, union: union<Int, Boolean>, lst: List<Int>, obj: A) -> result_1: Any
```

Comment:
Any identifier that is a keyword (e.g. `union`) must be escaped, by surrounding it with backticks.

Reply:
Where does the `result_1: Any` come from? The same applies for the other functions too.

Reply:
Currently they are created if a function / method has no return value as type hint. After discussing it with @lars-reimann we came to the conclusion to change the current functionality and remove the ```Any``` type hint information we create through the API and instead it should be inferred with the information that mypy provides. If there are no information, the field should be left empty.

Reply:
Created issue #34.